api/nakadi-event-bus-api.yaml

```diff
@@ -1177,8 +1177,7 @@ definitions:
         allOf:
           - $ref: '#/definitions/EventTypeSchema'
         description: |
-          The schema for this EventType. This is expected to be a json schema in yaml format (other formats
-          might be added in the future).
```

Comment:
@v-stepanov my mistake. There were so many small tasks going on that I confused myself with a local branch. https://github.com/zalando/nakadi/pull/238/commits/c126b06d857b099773ef395641bafb1c740befbd